Shaders/DisplayDepth.fx

```diff
@@ -8,251 +8,426 @@
 
 #include "ReShade.fxh"
 
+/**
+#ifdef __RESHADE__
+#undef __RESHADE__
+#define __RESHADE__ 40000
+#endif
+#ifdef ADDON_ADJUST_DEPTH
+#undef ADDON_ADJUST_DEPTH
+#endif
+**/
```

Comment:
This looks like testing left-overs that don't need to be in the final file.

Reply:
Yes. It exists for me and other translators.
I will keep it at least until the last commit before merging.